frontend/src/components/navbar/TopNavbar.vue

```diff
@@ -101,27 +127,61 @@
 </template>
 
 <script>
-import { mapActions, mapGetters } from 'vuex';
+import { mapActions, mapGetters, mapMutations } from 'vuex';
 import NavbarSearchPanel from './NavbarSearchPanel';
 import LanguageSelector from './LanguageSelector';
+import LikeReviewNotification from '@/components/notifications/LikeReviewNotification';
```

Comment:
LikeReviewNotification is duplicated 3 times ?

Reply:
now is not